public/stylesheet.css

```diff
@@ -634,21 +639,26 @@ body:not(.dark):not(.light) #color-scheme-system:not(:disabled) {
 }
 
 .grid-item__button {
+  fill: var(--color-button-active);
   border-top: 1px solid var(--color-grid-item-divider);
   background-color: var(--color-background);
-  color: var(--color-button-active);
   flex-grow: 1;
   font-family: var(--font-family-stylized);
   font-size: 0.75rem;
   font-weight: 600;
-  outline: none;
   padding: 8px;
   text-align: center;
 }
+
+.grid-item__button svg {
+  height: 1rem;
+  pointer-events: none;
+}
+
 .grid-item__button:focus,
 .grid-item__button:hover {
-  background: var(--color-button-hover);
-  color: var(--color-button-text-hover);
+  fill: var(--color-background);
```

Comment:
Did you intentionally change the color of "the content" (whether text or an SVG) from `var(--color-button-text-hover)` to `var(--color-background)`. I guess it looks fine but "color background" seem like a non-ideal choice for something that is not the background.